samples/misc/nif/readme.txt

```diff
@@ -42,15 +43,16 @@ running this program.
 - Under Windows or Linux
   type the following at the system command prompt:
  
-        nif <Database Name> <Time Variant View Name> <Normal View>
+        nif <Database Name> <Time Variant View Name> <Normal View> <json>

```

Comment:
Its shall be "json" instead of <json>, for my understanding it represents as user have to enter json string